### PR TITLE
Update customer-lockbox-requests.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about using PowerShell to approve/deny customer lockbox requests

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2483

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/compliance/customer-lockbox-requests.md [hunk 1]
```diff
@@ -97,6 +97,10 @@ You can turn on Customer Lockbox controls in the Microsoft 365 admin center. Whe
 
     ![Approve or deny Customer Lockbox requests](../media/CustomerLockbox8.png)
 
+> [!Note]
+> Use the Set-AccessToCustomerDataRequest cmdlet to approve, deny, or cancel Microsoft 365 customer lockbox requests that control access to your data by Microsoft support engineers. For more information, See [Set-AccessToCustomerDataRequest](https://docs.microsoft.compowershell/module/exchange/set-accesstocustomerdatarequest?view=exchange-ps)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-lockbox-requests.md]:
```suggestion
> Use the Set-AccessToCustomerDataRequest cmdlet to approve, deny, or cancel Microsoft 365 customer lockbox requests that control access to your data by Microsoft support engineers. For more information, see [Set-AccessToCustomerDataRequest](https://docs.microsoft.compowershell/module/exchange/set-accesstocustomerdatarequest?view=exchange-ps).
```

Comment (JohanFreelancer9) [reply microsoft-365/compliance/customer-lockbox-requests.md]: The link also seems to be incorrect and does not work. Please check.

File: microsoft-365/compliance/customer-lockbox-requests.md [hunk 2]
```diff
@@ -97,6 +97,10 @@ You can turn on Customer Lockbox controls in the Microsoft 365 admin center. Whe
 
     ![Approve or deny Customer Lockbox requests](../media/CustomerLockbox8.png)
 
+> [!Note]
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-lockbox-requests.md]:
```suggestion
> [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — COMMENTED** [review]
Please also confirm that the link is correct.

**Comment by msbemba** [comment]
> Please also confirm that the link is correct.

updated the link.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Pull request merged**